rental.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'book'
+require_relative 'class_person'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] I think here The rental is adding itself to the @book and @person achieving the 'belongs-to' association` so kindly assign both the book and person to the rental `book.rentals << self` `person.rentals << self` 💯🔥✨🙏🏻